schema.json

```diff
@@ -0,0 +1,72 @@
+{
+    "type" : "object",
+     "properties": {
+        "timestamp": {
+            "type": "string",
+	        "pattern": "^\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d+-\\d+$",
```

Comment:
Need to handle a "+" in the timezone offset as well for timestamps like:
```2020-07-30T14:53:17.457474+0000```
this should probably do it:
```
            "pattern": "^\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d+[+\\-]\\d+$",
```